src/interface.js

```diff
@@ -0,0 +1,54 @@
+$(document).ready( function() {
+  const game = new Game();
+  let rollNumber = 0;
+  let roll1;
+
+  $('button').on('click', function(e) {
+    roll = parseInt(e.target.innerHTML, 10);
+    rollNumber += 1;
+
+    //   to disable button  $("#btn").attr("disabled", true);
+    if ((rollNumber%2)==0) {
+      const frame = new Frame(roll1, roll);
+      try {
+        game.play(frame);
+        updateTable();
+      } catch (err) {
+        alert(err.message);
```

Comment:
Great use of try/catch - I wanted to implement it but couldn't figure out so it's useful to see how it's used